### DISCUSSION TITLE
Internal Server Error when running web-application-example

### DISCUSSION THREAD

**Discussion by subsurfaceiodev** [fief-dev/fief#241] [Q&A] [answered]
## Describe the bug

We get an **Internal Server Error** when running https://docs.fief.dev/integrate/python/fastapi/#web-application-example. Traceback shows error originated from:
[fief_client/client.py] `signed_id_token = jwt.JWT(jwt=id_token_claims, algs=["RS256"], key=jwks)`

--> TypeError: Expected JWS, got JWE

## To Reproduce

web-application-example, as stated above

## Expected behavior

No error

## Configuration

* Cloud or self-hosted: Cloud

## Additional context
https://docs.fief.dev/integrate/python/fastapi/#api-example runs fine with no errors

**Comment by fief-bailiff** [comment]
Hail, @subsurfaceiodev 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment, marked as answer]
It looks like you enabled [ID Token encryption](https://docs.fief.dev/going-further/id-token-encryption/) on your client but didn't pass the encryption key. So two solutions:

1. You want to keep ID Token encryption. In this case, follow the instructions here: https://docs.fief.dev/going-further/id-token-encryption/#enable-id-token-encryption If you didn't copy/paste your encryption key the first time, you'll have to regenerate it.
2. If you don't want to keep ID Token encryption, you can [create a new client](https://docs.fief.dev/admin-dashboard/clients/#create-a-new-client).

**Comment by subsurfaceiodev** [comment]
That was it! Thanks so much
